quisp/modules/QNIC/StationaryQubit/logicalStationaryQubit.h

```diff
@@ -0,0 +1,100 @@
+
+
+
+#ifndef QUISP_MODULES_LOGICAL_STATIONARYQUBIT_H_
+#define QUISP_MODULES_LOGICAL_STATIONARYQUBIT_H_
+
+#include "StationaryQubit.h"
+#include <vector>
+
+using namespace Eigen;
+using namespace omnetpp;
+using namespace quisp::messages;
+
+namespace quisp {
+    namespace modules {
+
+        typedef std::complex<double> Complex;
+
+        typedef struct _logical_quantum_state{
+            MatrixXcd state_in_density_matrix;
+            VectorXcd state_in_ket;
+        } logical_quantum_state;
+/*
+        typedef struct _measurement_outcome{
+            char basis;
+            bool outcome_is_plus;
+        }measurement_outcome;
+*/
+        typedef struct _logical_measurement_operator{
```

Comment:
you don't need to use `typedef ~` style.
it's ok like this
```cpp
struct LogicalMeasurementOperator {
  MatrixXcd plus;            
  MatrixXcd minus;            
  VectorXcd plus_ket;            
  VectorXcd minus_ket;            
  char basis;
} ;
```
https://godbolt.org/z/aberedT71